code/modules/admin/verbs/lightning_strike.dm

```diff
@@ -65,7 +65,9 @@
 	var/sound = get_sfx("thunder")
 	for(var/mob/M in player_list)
 		if((P && M.z in P.expected_z_levels) || M.z == T.z)
-			M.playsound_local(get_turf(M), soundin = sound, vol = 70, vary = FALSE, is_global = TRUE)
+			if(!(M && M.is_preference_enabled(/datum/client_preference/play_ambiance)))	return
```

Comment:
is it really supposed to be fucking "ambiance"

Reply:
cannot make this up, its spelled 2 different ways in the code. this is the correct way for functionality.